projects/plugins/jetpack/class.jetpack.php

```diff
@@ -3313,7 +3314,7 @@ public function throw_error_on_activate_plugin( $plugin ) {
 
 			if ( $throw ) {
 				/* translators: Plugin name to deactivate. */
-				trigger_error( sprintf( esc_html__( 'Jetpack contains the most recent version of the old &#8220;%1$s&#8221; plugin.', 'jetpack' ), 'WordPress.com Stats' ), E_USER_ERROR ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
+				throw new RuntimeException( sprintf( __( 'Jetpack contains the most recent version of the old “%1$s” plugin.', 'jetpack' ), 'WordPress.com Stats' ) );
```

Comment:
Re the unescaping, see [this issue elsewhere](https://href.li?https://github.com/WordPress/WordPress-Coding-Standards/issues/2374) for why escaping exception messages at throw time is the wrong thing to do.